packages/shared/ReactDOMFrameScheduling.js

```diff
@@ -79,17 +90,21 @@ if (!ExecutionEnvironment.canUseDOM) {
   var frameDeadlineObject;
   if (hasNativePerformanceNow) {
     frameDeadlineObject = {
+      didTimeout: false,
```

Comment:
Will you actually use this flag? You'll know based on time at the point you get called.

Reply:
Nah it's not used by React. I added in anticipation of pulling this out into a polyfill.